api/app/src/external/commonwell/link/create.ts

```diff
@@ -30,37 +32,41 @@ export const create = async (
 
     if (cwPersonId) {
       await reset(patient, organization);
-
-      await setCommonwellId({
-        patientId: patient.id,
-        cxId: patient.cxId,
-        commonwellPatientId: cwPatientId,
-        commonwellPersonId: undefined,
-      });
```

Comment:
This was not necessary since its in the reset function

Reply:
I thought this was here so we could remove the `personId` from our local DB to match the fact the patient is not linked to that person anymore.